frontend/tsconfig.json

```diff
@@ -26,13 +26,14 @@
         "./*"
       ]
     },
-    "target": "ES6"
+    "target": "ES2017"
```

Comment:
Was this intended? Something insert this automatically for me in the past.

Reply:
Hmm I'm not sure what it's from

Reply:
I can't remember what I was doing. I believe it was during the next.js update so could have been any number of things.
It looks like ES6 was added only 2 days ago. I'd be tempted to remove it. We use typescript 5 so it should default to ESNEXT.

Reply:
Ah okay. It was added 2 days ago, and there was no `target` property at all before that. 

I just tried removing it and it got re-added (as ES2017) when I ran `next dev` (via `npm run dev`)

Reply:
`next lint` does the same

Reply:
changed this back to what it was